src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTest.php

```diff
@@ -52,6 +52,9 @@
 
     abstract protected function loadFromFile(ContainerBuilder $container, $file);
 
+    /**
+     * @group legacy
+     */
     public function testFormCsrfProtection()
```

Comment:
I don't think form csrf is actually deprecated. It only uses the a config with templating enabled. So the test needs to be changed to keep this form csrf tested.